Documentation/Functions/Stdwrap/Index.rst

```diff
@@ -1024,22 +1024,25 @@ bytes
          integer /stdWrap
 
    Description
-         Will format the input (an integer) as bytes: bytes, kb, mb
+         Will format the input (an integer) as bytes: bytes, kb, mb, …
 
          **Available sub-properties:**
 
          **labels**:
          If you add a value for the property "labels" you can alter the default
-         suffixes. Labels for bytes, kilo, mega and giga are separated by
-         vertical bar (\|) and possibly encapsulated in "". E.g.: " \| K\| M\| G"
-         (which is the default value).
-
+         suffixes. Accept a preset keyword or labels for bytes, kilo, mega, giga
```

Comment:
I don't understand this sentence (?): "Accept ..." What do you mean?

Reply:
The field accept either of a preset keyword or a list of label. Should I try reformulate? Any suggestion?

Reply:
Sure! You should exactly have written that :-) (The field accepts either ...) It's ok, you don't have to send an update. I'll make it fit and check it in tomorrow.